lib/opFns.js

```diff
@@ -555,8 +555,12 @@ module.exports = {
         return
       }
       if (!exists) {
-        // can't wrap because we are in a callback
-        runState.gasLeft.isub(new BN(fees.callNewAccountGas.v))
+        try {
+          subGas(runState, new BN(fees.callNewAccountGas.v))
+        } catch (e) {
+          done(e.error)
```

Comment:
Right, this commit log was a bit misleading. It fixes a bug when there wasn't sufficient gas left.